lib/MusicBrainz/Server/Data/Release.pm

```diff
@@ -810,7 +816,7 @@ sub can_merge {
             @old_ids, $new_id);
 
         if ($mediums_differ) {
-            $opts->{_cannot_merge_reason} = N_l('The track counts on at least one set of corresponding mediums do not match.');
+            $opts->{_cannot_merge_reason} = $RELEASE_MERGE_ERRORS{medium_track_counts};
             return 0;
         }
```

Comment:
This should probably be `if ($medium_track_counts_differ)` then? `$mediums_differ` could mean anything :)

Reply:
Technically this error is also triggered if the target release has less mediums than the source release (but it lets you merge in the other direction, I don't recall why). So the message is also wrong in that case. Should I add a separate message for that case, or should we allow that type of merge if possible?

Reply:
The other direction is needed so you can merge an orphaned CD 1 out of a set for example into the set :) I'd make it a different message for the big into small merge, but still block it - it'd be hard to understand the user's expected result in that case IMO.

Reply:
Right, I mean I knew why it was needed but didn't recall why it only allowed one direction and not the other. But if it keeps the intent clear I'll leave it blocked & and come up with a suitable message for that.

Reply:
This should be fixed by e17884f :)